README.md

```diff
@@ -9,15 +9,23 @@ The machine learning part of this project refers to the legendary [ggml.cpp](htt
 
 `pip install -r requirements.txt`
 
-## MNIST
-
-1. Train the AI model. See `examples/mnist/trainning/mnist.ipynb`
-2. Convert the AI model into GGML using `examples/mnist/convert-h5-to-ggml.py`
-3. Build the AI inference engine for MIPS
-`cd examples/mnist_mips && ./build`
-``
-
-## LLAMA
-
-Support LLaMA 1 and LLaMA 2 (7B and 13B).
-Please refer to `examples/llama/README.md` and download the model first.
\ No newline at end of file
+## Linear Model
```

Comment:
Please do not delete the previous content in README. Just add the new content at the end.